code/extensions/UserDefinedFormFreshdeskExtension.php

```diff
@@ -12,4 +12,17 @@ public function updateCMSFields(FieldList $fields)
         $fields->addFieldToTab('Root.FormOptions', new CheckboxField('ExportToFreshdesk', 'Export as a Freshdesk ticket on submit'));
         $fields->addFieldToTab('Root.FormOptions', new TextareaField('FreshdeskDescription', 'Freshdesk ticket description:'));
     }
+
+    public function validate(ValidationResult $validationResult)
```

Comment:
My opinion is this is a wrong place to put it, because:

* the concept of validation applies to the data of this specific DataObject. It's conceptually incorrect to use it for validating some random value someone posted on the Internet :-)
* there is only a weak relationship between the admin saving the form and requiring the constant to be set. For example I can imagine someone restoring a snapshot with the box ticked, and bypassing this check.

Maybe remove the check completely, and only make sure ->call fails gracefully and reports the exception gracefully (and sends us alerts via grumpy etc)